.github/workflows/ci.canary.yml

```diff
@@ -54,52 +54,17 @@ jobs:
         env:
           POLYGON_MAINNET_PROVIDER_URL: ${{ secrets.POLYGON_MAINNET_PROVIDER_URL }}
 
-  subgraph-test:
-    name: Run test of subgraph on dev branch
-
+  test-subgraph:
+    uses: superfluid-finance/protocol-monorepo/.github/workflows/call.setup-deploy-and-test-local-subgraph.yml@dev
```

Comment:
I would add these dependent "call workflow" yml files to the "paths" dependents list.